src/osx/core/dcmemory.cpp

```diff
@@ -65,14 +65,18 @@ void wxMemoryDCImpl::DoSelect( const wxBitmap& bitmap )
 {
     if ( m_selected.IsOk() )
     {
-        m_selected.EndRawAccess() ;
+        m_selected.SetSelectedInto(NULL);
         wxDELETE(m_graphicContext);
     }
 
     m_selected = bitmap;
     if (m_selected.IsOk())
     {
-        m_selected.BeginRawAccess() ;
+        wxASSERT_MSG( !bitmap.GetSelectedInto() ||
+                     (bitmap.GetSelectedInto() == GetOwner()),
+                     wxT("Bitmap is selected in another wxMemoryDC, delete the first wxMemoryDC or use SelectObject(NULL)") );
```

Comment:
Please avoid using `wxT()` in the new code. In all but the most performance-critical places it's ok to omit it entirely, otherwise you can use `wxS()` to avoid an extra conversion, but `wxT()` is never useful.

Reply:
done